### ISSUE TITLE
Support for Common Props

### ISSUE THREAD

**Issue by nikeee** [skarpdev/dotnet-version-cli#85]
I have the following setup:

`Common.props`
```xml
<Project>
  <PropertyGroup>
    <Version>2.0.0</Version>
    <!-- ... a lot more ... -->
  </PropertyGroup>
</Project>
```

And multiple csproj files like this:
`a/A.csproj`:
```xml
<Project Sdk="Microsoft.NET.Sdk">
  <Import Project="..\Common.props" />
</Project>
```

Changing the version in the `Common.props` would affect all projects that import this file. This basically allows us that we don't have to maintain `n` separate csproj file versions.

Trying to use `dotnet version` with that, I get this:
```sh
$ dotnet version -f a/A.csproj
dotnet-version-cli version 2.1.3.0
Project version is: 
	0.0.0
```
(which is probaby because the `Import` isn't recognized)

When using `Common.props` as a "csproj" file, I get this error:
```sh
$ dotnet version -f Common.props
dotnet-version-cli version 2.1.3.0
ERR Something went completely haywire, developer zen:
	Not a directory STACK: 
   at System.IO.Enumeration.FileSystemEnumerator`1.CreateDirectoryHandle(String path, Boolean ignoreNotFound)
   at System.IO.Enumeration.FileSystemEnumerator`1.Init()
   at System.IO.Enumeration.FileSystemEnumerator`1..ctor(String directory, Boolean isNormalized, EnumerationOptions options)
   at System.IO.Enumeration.FileSystemEnumerable`1..ctor(String directory, FindTransform transform, EnumerationOptions options, Boolean isNormalized)
   at System.IO.Enumeration.FileSystemEnumerableFactory.UserFiles(String directory, String expression, EnumerationOptions options)
   at System.IO.Directory.InternalEnumeratePaths(String path, String searchPattern, SearchTarget searchTarget, EnumerationOptions options)
   at System.IO.Directory.EnumerateFiles(String path)
   at Skarp.Version.Cli.CsProj.FileSystem.DotNetFileSystemProvider.List(String path) in C:\projects\dotnet-version-cli\src\CsProj\FileSystem\DotNetFileSystemProvider.cs:line 15
   at Skarp.Version.Cli.CsProj.ProjectFileDetector.FindAndLoadCsProj(String bootstrapPath) in C:\projects\dotnet-version-cli\src\CsProj\ProjectFileDetector.cs:line 44
   at Skarp.Version.Cli.VersionCli.DumpVersion(VersionCliArgs args) in C:\projects\dotnet-version-cli\src\VersionCli.cs:line 116
   at Skarp.Version.Cli.Program.<>c__DisplayClass1_0.<Main>b__0() in C:\projects\dotnet-version-cli\src\Program.cs:line 89
```

It would be cool if the latter would work. If I'm seeing this correctly, it would also be easier than implementing the former because we don't have to resolve import statements and merge their contents into the current project definitions.

Having this would also make a `--recursive` option unnecessary for me, since I can just update one specific version that is included in all relevant csproj files (see #68).

The above error seems to originate from the csproj check:
https://github.com/skarpdev/dotnet-version-cli/blob/master/src/CsProj/ProjectFileDetector.cs#L38
Changing the file extension to `csproj` works.

**Comment by nikeee** [comment]
I found out that MS typically uses the `.targets` file extension for that (at least they do it in their docs): https://docs.microsoft.com/en-us/visualstudio/msbuild/import-element-msbuild?view=vs-2019

Would this change be something you'd merge?
https://github.com/skarpdev/dotnet-version-cli/compare/master...nikeee:master

**Comment by nikeee** [comment]
> That should probably be

Done. PR is out

**Comment by nover** [comment]
@nikeee merged - currently rolling to nuget as `2.2.0`
